target/linux/mediatek/image/Makefile

```diff
@@ -27,15 +27,9 @@ define Device/Default
   IMAGES := sysupgrade.bin
   IMAGE/sysupgrade.bin := append-kernel | pad-to 128k | append-rootfs | pad-rootfs | append-metadata
   SUPPORTED_DEVICES := $(subst _,$(comma),$(1))
-ifeq ($(SUBTARGET),mt7623)
-  KERNEL_NAME := zImage
```

Comment:
Please also remove the redundant unused KERNEL_NAME six lines above.

Reply:
done.